### ISSUE TITLE
npm deprecate causes E500

### ISSUE THREAD

**Issue by kumavis** [npm/npm#15619]
#### I'm opening this issue because:

  - [x] npm is crashing.

#### What's going wrong?
npm deprecate is failing with E500

#### How can the CLI team reproduce the problem?

```
╭─kumavis@xyzs-MBP  ~/dev/ethereumjs-tx ‹node-v6.3.1›  (master) 
╰─$ npm deprecate "ethereumjs-tx@1.2.1" "replay protection hash fixed in v1.2.4"                                                                                                 1 ↵
npm ERR! Darwin 16.3.0
npm ERR! argv "/Users/kumavis/.nvm/versions/node/v6.3.1/bin/node" "/Users/kumavis/.nvm/versions/node/v6.3.1/bin/npm" "deprecate" "ethereumjs-tx@1.2.1" "replay protection hash fixed in v1.2.4"
npm ERR! node v6.3.1
npm ERR! npm  v4.1.2
npm ERR! code E500

npm ERR! "ethereumjs-tx status = 403" : ethereumjs-tx
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/kumavis/dev/ethereumjs-tx/npm-debug.log
```

### supporting information:

```
╭─kumavis@xyzs-MBP  ~/dev/ethereumjs-tx ‹node-v6.3.1›  (master) 
╰─$ npm -v
4.1.2
╭─kumavis@xyzs-MBP  ~/dev/ethereumjs-tx ‹node-v6.3.1›  (master) 
╰─$ node -v 
v6.3.1
╭─kumavis@xyzs-MBP  ~/dev/ethereumjs-tx ‹node-v6.3.1›  (master) 
╰─$ npm config get registry
https://registry.npmjs.org/
```
 - Windows, OS X/macOS, or Linux?:
macOs `Darwin xyzs-MBP.domain 16.3.0 Darwin Kernel Version 16.3.0: Thu Nov 17 20:23:58 PST 2016; root:xnu-3789.31.2~1/RELEASE_X86_64 x86_64`

reproduced without vpn

**Comment by kumavis** [comment]
`npm-debug.log`: https://gist.github.com/kumavis/ecc6bae17b05b0630cc51bb4aee694a8

**Comment by dzonatan** [comment]
@kumavis Were you able to solve this issue?

**Comment by catdad** [comment]
I'm having this same issue. I had v0.2.0 of my module. Then, I published v0.3.0 which was totally broken. So I republished the older version as v0.4.0 until I am able to fix it, and wanted to deprecate v0.3.0 so users get a message if they try to install it. But I get this error and am unable to deprecate it.

node -v: 6.4.0
npm -v: 3.10.3

Here is the command I am trying to run:

```bash
npm deprecate gulp-each@0.3.0 "critical bug in v0.3.x fixed in v0.4.0"
```

Here is the console output:

```
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "deprecate" "gulp-each@0.3.1" "critical bug in v0.3.x fixed in v0.4.0"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3
npm ERR! code E500

npm ERR! "gulp-each status = 403" : gulp-each
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     D:\Git\gulp-each\npm-debug.log
```

Here is the content of the npm log:

```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'deprecate',
1 verbose cli   'gulp-each@0.3.1',
1 verbose cli   'critical bug in v0.3.x fixed in v0.4.0' ]
2 info using npm@3.10.3
3 info using node@v6.4.0
4 silly mapToRegistry name gulp-each
5 silly mapToRegistry using default registry
6 silly mapToRegistry registry https://registry.npmjs.org/
7 silly mapToRegistry data Result {
7 silly mapToRegistry   raw: 'gulp-each',
7 silly mapToRegistry   scope: null,
7 silly mapToRegistry   escapedName: 'gulp-each',
7 silly mapToRegistry   name: 'gulp-each',
7 silly mapToRegistry   rawSpec: '',
7 silly mapToRegistry   spec: 'latest',
7 silly mapToRegistry   type: 'tag' }
8 silly mapToRegistry uri https://registry.npmjs.org/gulp-each
9 verbose get GET as part of write; not caching result
10 verbose request uri https://registry.npmjs.org/gulp-each?write=true
11 verbose request no auth needed
12 info attempt registry request try #1 at 11:43:26 PM
13 verbose request id 729a44873653827f
14 http request GET https://registry.npmjs.org/gulp-each?write=true
15 http 200 https://registry.npmjs.org/gulp-each?write=true
16 verbose headers { 'content-type': 'application/json',
16 verbose headers   server: 'nginx/1.10.1',
16 verbose headers   'last-modified': 'Tue, 21 Feb 2017 04:31:56 GMT',
16 verbose headers   etag: 'W/"58abc2bc-3e33"',
16 verbose headers   'content-encoding': 'gzip',
16 verbose headers   'cache-control': 'max-age=0',
16 verbose headers   'transfer-encoding': 'chunked',
16 verbose headers   'accept-ranges': 'bytes',
16 verbose headers   date: 'Tue, 21 Feb 2017 04:43:35 GMT',
16 verbose headers   via: '1.1 varnish',
16 verbose headers   age: '0',
16 verbose headers   connection: 'keep-alive',
16 verbose headers   'x-served-by': 'cache-jfk8135-JFK',
16 verbose headers   'x-cache': 'MISS',
16 verbose headers   'x-cache-hits': '0',
16 verbose headers   'x-timer': 'S1487652215.695525,VS0,VE96',
16 verbose headers   vary: 'Accept-Encoding' }
17 silly get cb [ 200,
17 silly get   { 'content-type': 'application/json',
17 silly get     server: 'nginx/1.10.1',
17 silly get     'last-modified': 'Tue, 21 Feb 2017 04:31:56 GMT',
17 silly get     etag: 'W/"58abc2bc-3e33"',
17 silly get     'content-encoding': 'gzip',
17 silly get     'cache-control': 'max-age=0',
17 silly get     'transfer-encoding': 'chunked',
17 silly get     'accept-ranges': 'bytes',
17 silly get     date: 'Tue, 21 Feb 2017 04:43:35 GMT',
17 silly get     via: '1.1 varnish',
17 silly get     age: '0',
17 silly get     connection: 'keep-alive',
17 silly get     'x-served-by': 'cache-jfk8135-JFK',
17 silly get     'x-cache': 'MISS',
17 silly get     'x-cache-hits': '0',
17 silly get     'x-timer': 'S1487652215.695525,VS0,VE96',
17 silly get     vary: 'Accept-Encoding' } ]
18 verbose get saving gulp-each to C:\Users\Kiril\AppData\Roaming\npm-cache\registry.npmjs.org\gulp-each_3Fwrite_3Dtrue\.cache.json
19 verbose correctMkdir C:\Users\Kiril\AppData\Roaming\npm-cache correctMkdir not in flight; initializing
20 verbose request uri https://registry.npmjs.org/gulp-each
21 verbose request sending authorization for write operation
22 info attempt registry request try #1 at 11:43:28 PM
23 http request PUT https://registry.npmjs.org/gulp-each
24 http 500 https://registry.npmjs.org/gulp-each
25 verbose headers { 'content-type': 'application/json',
25 verbose headers   'cache-control': 'max-age=0',
25 verbose headers   'content-length': '54',
25 verbose headers   'accept-ranges': 'bytes',
25 verbose headers   date: 'Tue, 21 Feb 2017 04:43:37 GMT',
25 verbose headers   via: '1.1 varnish',
25 verbose headers   connection: 'keep-alive',
25 verbose headers   'x-served-by': 'cache-jfk8135-JFK',
25 verbose headers   'x-cache': 'MISS',
25 verbose headers   'x-cache-hits': '0',
25 verbose headers   'x-timer': 'S1487652215.846378,VS0,VE1220' }
26 info retry will retry, error on last attempt: Error: "gulp-each status = 403" : gulp-each
27 info attempt registry request try #2 at 11:43:39 PM
28 http request PUT https://registry.npmjs.org/gulp-each
29 http 500 https://registry.npmjs.org/gulp-each
30 verbose headers { 'content-type': 'application/json',
30 verbose headers   'cache-control': 'max-age=0',
30 verbose headers   'content-length': '54',
30 verbose headers   'accept-ranges': 'bytes',
30 verbose headers   date: 'Tue, 21 Feb 2017 04:43:48 GMT',
30 verbose headers   via: '1.1 varnish',
30 verbose headers   connection: 'keep-alive',
30 verbose headers   'x-served-by': 'cache-jfk8135-JFK',
30 verbose headers   'x-cache': 'MISS',
30 verbose headers   'x-cache-hits': '0',
30 verbose headers   'x-timer': 'S1487652227.090406,VS0,VE1235' }
31 info retry will retry, error on last attempt: Error: "gulp-each status = 403" : gulp-each
32 info attempt registry request try #3 at 11:44:40 PM
33 http request PUT https://registry.npmjs.org/gulp-each
34 http 500 https://registry.npmjs.org/gulp-each
35 verbose headers { 'content-type': 'application/json',
35 verbose headers   'cache-control': 'max-age=0',
35 verbose headers   'content-length': '54',
35 verbose headers   'accept-ranges': 'bytes',
35 verbose headers   date: 'Tue, 21 Feb 2017 04:44:49 GMT',
35 verbose headers   via: '1.1 varnish',
35 verbose headers   connection: 'keep-alive',
35 verbose headers   'x-served-by': 'cache-jfk8135-JFK',
35 verbose headers   'x-cache': 'MISS',
35 verbose headers   'x-cache-hits': '0',
35 verbose headers   'x-timer': 'S1487652288.348649,VS0,VE1384' }
36 verbose request invalidating C:\Users\Kiril\AppData\Roaming\npm-cache\registry.npmjs.org\gulp-each on PUT
37 verbose stack Error: "gulp-each status = 403" : gulp-each
37 verbose stack     at makeError (C:\Users\Kiril\AppData\Roaming\nvm\v6.4.0\node_modules\npm\node_modules\npm-registry-client\lib\request.js:264:12)
37 verbose stack     at CachingRegistryClient.<anonymous> (C:\Users\Kiril\AppData\Roaming\nvm\v6.4.0\node_modules\npm\node_modules\npm-registry-client\lib\request.js:252:14)
37 verbose stack     at Request._callback (C:\Users\Kiril\AppData\Roaming\nvm\v6.4.0\node_modules\npm\node_modules\npm-registry-client\lib\request.js:172:14)
37 verbose stack     at Request.self.callback (C:\Users\Kiril\AppData\Roaming\nvm\v6.4.0\node_modules\npm\node_modules\request\request.js:200:22)
37 verbose stack     at emitTwo (events.js:106:13)
37 verbose stack     at Request.emit (events.js:191:7)
37 verbose stack     at Request.<anonymous> (C:\Users\Kiril\AppData\Roaming\nvm\v6.4.0\node_modules\npm\node_modules\request\request.js:1067:10)
37 verbose stack     at emitOne (events.js:101:20)
37 verbose stack     at Request.emit (events.js:188:7)
37 verbose stack     at IncomingMessage.<anonymous> (C:\Users\Kiril\AppData\Roaming\nvm\v6.4.0\node_modules\npm\node_modules\request\request.js:988:12)
38 verbose statusCode 500
39 verbose pkgid gulp-each
40 verbose cwd D:\Git\gulp-each
41 error Windows_NT 10.0.14393
42 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "deprecate" "gulp-each@0.3.1" "critical bug in v0.3.x fixed in v0.4.0"
43 error node v6.4.0
44 error npm  v3.10.3
45 error code E500
46 error "gulp-each status = 403" : gulp-each
47 error If you need help, you may report this error at:
47 error     <https://github.com/npm/npm/issues>
48 verbose exit [ 1, true ]
```

**Comment by dzonatan** [comment]
Duplicate of #15450
